### PR TITLE
add PullRequestCheckoutCommand

### PR DESCRIPTION
**Pull request by sstok**
|Q            |A   |
|---          |--- |
|Fixed Tickets|#463|
|License      |MIT |

### CODE REVIEW

File: src/Command/PullRequest/PullRequestCheckoutCommand.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+class PullRequestCheckoutCommand extends BaseCommand implements GitRepoFeature, GitFolderFeature
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('pull-request:checkout')
+            ->setDescription('Checkouts a pull request as local branch')
```
Comment (cordoval) [inline src/Command/PullRequest/PullRequestCheckoutCommand.php]: checks out

File: src/Command/PullRequest/PullRequestCheckoutCommand.php [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            ->addArgument('pr_number', InputArgument::REQUIRED, 'Pull Request number to be checked-out')
+            ->setHelp(
+                <<<EOF
+The <info>%command.name%</info> command is used to checkout a pull-request from the organization.
```
Comment (cordoval) [inline src/Command/PullRequest/PullRequestCheckoutCommand.php]: to check a pull-request out

File: src/Command/PullRequest/PullRequestCheckoutCommand.php [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+When the branch already exists Gush will check if the remote-upstream is the same
+as the source organization of the pull-request and checkout the local branch instead.
```
Comment (cordoval) [inline src/Command/PullRequest/PullRequestCheckoutCommand.php]: and check out

**Comment by sstok** [comment]
Resolved the spelling issues. I wonder why Travis is hanging :confused:

**Comment by cordoval** [comment]
closing in favor of #465

**Pull request closed without merging**